atlasdb-jepsen-tests/src/jepsen/atlasdb/timestamp.clj

```diff
@@ -109,5 +71,5 @@
                                      (gen/sleep 85)
                                      {:type :info, :f :stop}])))
                     (gen/time-limit 360))
-    :db (create-server)
+    :db (timelock/create-db)
```

Comment:
You're quite right! I removed those waits during the refactor because I think they are unnecessary. As soon as the timelock server is listening on the port, it should be ready to go. 

I'm running the Jepsen test twenty times on internal CI to confirm that this doesn't cause any flakes: if it does, I'll re-add back in the extra two waits.